### PR TITLE
109) Fix assorted issues in LivingEntity

### PR DESCRIPTION
**Pull request by tkgdhughes**
### Description

I wasn't sure whether to push this or not since it is really just tidying things up, but since I am submitting more or else everything, I thought I would give you the option to take it! This is just three small changes to tidy up some code in livingentity.cpp.

### CODE REVIEW

File: dev/Code/CryEngine/CryPhysics/livingentity.cpp [hunk 1]
```diff
@@ -198,6 +198,9 @@ CLivingEntity::CLivingEntity(CPhysicalWorld* pWorld, IGeneralMemoryHeap* pHeap)
     m_dtRequested = 0;
 
     m_bReleaseGroundColliderWhenNotActive = 1;
+    m_timeStepFull = 0.0f;
+    m_timeStepPerformed = 0.0f;
+    m_iLastGroundColliderPart = -1;
```
Comment (tkgdhughes) [inline dev/Code/CryEngine/CryPhysics/livingentity.cpp]: Initialise variables which were missed.

File: dev/Code/CryEngine/CryPhysics/livingentity.cpp [hunk 2]
```diff
@@ -793,7 +796,6 @@ int CLivingEntity::GetStatus(pe_status* _status) const
         status->bSquashed = m_bSquashed;
         return 1;
     }
-    else
```
Comment (tkgdhughes) [inline dev/Code/CryEngine/CryPhysics/livingentity.cpp]: This else seem misplaced based on the structure of the rest of the code.

File: dev/Code/CryEngine/CryPhysics/livingentity.cpp [hunk 3]
```diff
@@ -1373,18 +1375,7 @@ int CLivingEntity::SetStateFromSnapshot(TSerialize ser, int flags)
...
-                setpos.pos = helper.pos + (m_pos - helper.pos).GetNormalized() * MAX_DIFFERENCE;
-            else
-                setpos.pos = m_pos + (helper.pos - m_pos) * distance/MAX_DIFFERENCE*0.033f;*/
-            setpos.pos = helper.pos;
-        }
-        else
-        {
-            setpos.pos = helper.pos;
-        }
+        setpos.pos = helper.pos;
```
Comment (tkgdhughes) [inline dev/Code/CryEngine/CryPhysics/livingentity.cpp]: Both sides of the `if/else` were doing exactly the same thing.

**Comment by AMZN-alexpete** [comment]
Thanks @tkgdhughes, these changes have been approved and will be in a future version of Lumberyard.  Cheers!

**Pull request closed without merging**